tests/pages/dashboardPage.js

```diff
@@ -841,6 +873,15 @@ module.exports = {
     }
   },
 
+  openGraphDropdownMenu(metric) {
+    I.seeElement(this.graphsLocator(metric));
+    I.click(this.graphsLocator(metric));
+  },
+
+  graphDropdownMenu(metric) {
+    return this.graphsLocator(metric) + `/..//ul[contains(@class, 'dropdown-menu')]`;
```

Comment:
what exactly is this menu?

Reply:
removed graphDropdownMenu and added a shorter locator in test 1070